app/models/parameter.rb

```diff
@@ -16,9 +16,7 @@ class Parameter < ActiveRecord::Base
   default_scope -> { order("parameters.name") }
 
   before_validation :strip_whitespaces
-  after_initialize :set_priority, :ensure_reference_nil
-
-  PRIORITY = {:common_parameter => 0, :domain_parameter => 1, :os_parameter => 2, :group_parameter => 3, :host_parameter => 4}
+  after_initialize :ensure_reference_nil
```

Comment:
For the record priority should be removed from the rest of the parameters but it should not be part of this pr. After this is merged I'll open an issue about it.